mutation/mutation_compactor.hh

```diff
@@ -289,7 +292,9 @@ private:
         if (_max_purgeable == api::missing_timestamp) {
             _max_purgeable = _get_max_purgeable(*_dk, is_shadowable);
         }
-        return t.timestamp < _max_purgeable;
+        auto ret = t.timestamp < _max_purgeable;
+        mclog.debug("can_gc: t={} is_shadowable={} max_purgeable={}: ret={}", t, is_shadowable, _max_purgeable, ret);
```

Comment:
This is a general question I always ask myself, it's not only for this patch:
Does it really help when developers leave behind various debug messages in the code that helped them write this code? Is there a chance that this specific printout will ever help a future developer? Or maybe the future developer will find himself adding a slightly different printout anyway?

Reply:
I needed it to understand why the unit test (and then the cql-pytest) don't work as I expected, so I assume they could help in the field if tombstone garbage collection doesn't work as expected either.